### ISSUE TITLE
construction errors with CenterColumnStudyReactor

### ISSUE THREAD

**Issue by Shimwell** [ukaea/paramak#338]
With some arguments settings the CenterColumnStudyReactor fails to produce geometry

The following model doesn't work over here

```
import paramak            
my_reactor = paramak.CenterColumnStudyReactor(
    inner_bore_radial_thickness=50,
    inboard_tf_leg_radial_thickness=100,
    center_column_shield_radial_thickness_mid=50,
    center_column_shield_radial_thickness_upper=70,
    inboard_firstwall_radial_thickness=20,
    divertor_radial_thickness=50,
    inner_plasma_gap_radial_thickness=50,
    plasma_radial_thickness=100,
    outer_plasma_gap_radial_thickness=50,
    center_column_arc_vertical_thickness=200,
    plasma_high_point=(350, 0),
    plasma_gap_vertical_thickness=20,
    rotation_angle=360,
)
```

Perhaps this is a good time to rewrite the model so that is avoids the use of plasma_high_point which can be difficult for users to set correctly

**Comment by Shimwell** [comment]
Getting some more construction errors randomly on the tests
https://app.circleci.com/pipelines/github/ukaea/paramak/1091/workflows/da9b52c2-c283-4302-bd20-05c23d2fe8dc/jobs/1123

**Comment by RemDelaporteMathurin** [comment]
Is this related to #445 ?

**Comment by Shimwell** [comment]
I think I have figured this one out and have a fix in the works. Basically the plasma_high_point is a tricky way to go about building a reactor

**Comment by RemDelaporteMathurin** [comment]
@Shimwell this has been fixed in #470. Closing this
